services/brig/src/Brig/API/User.hs

```diff
@@ -258,6 +272,12 @@ createUser new@NewUser {..} = do
     handleTeam (Just (NewTeamCreator t)) _ = (Just t,Nothing,) <$> (Just . Id <$> liftIO nextRandom)
     handleTeam (Just (NewTeamMemberSSO tid)) _ = pure (Nothing, Nothing, Just tid)
     handleTeam Nothing _ = return (Nothing, Nothing, Nothing)
+
+    removeTTL account = do
+      let pw = Nothing
+      let activated = False
+      lift $ Data.insertAccount account Nothing pw activated (Just 0)
```

Comment:
I think I had doubts if `insertAccount` would overwrite everything, but during pairing I think we figured out that it does?

Reading back the entire row and writing it would not solve the problems with nulls anyway:
1. read whole row for which the `handle` is null and write it back with a TTL. Since `handle` is null on the write no TTL is set for this column.
2. set handle to non-null without TTL
3. the row will not expire

Reply:
resolved, as we decided not to use TTL anymore to clear pending users